storage/innobase/xtrabackup/doc/source/howtos/backup_verification.rst

```diff
@@ -47,11 +47,11 @@ Running the *pt-table-checksum* on the source will create ``percona`` database w
 
 If all the values in the ``DIFFS`` column are 0 that means that backup is consistent with the current setup.
 
-In case backup wasn't consistent  *pt-table-checksum* should spot the difference and point to the table that doesn't match. Following example shows adding new user on the backed up replica in order to simulate the inconsistent backup: ::
+In case backup wasn't consistent  *pt-table-checksum* should spot the difference and point to the table that doesn't match. The following example adds a new user on the backed up replica in order to simulate the inconsistent backup: ::
```

Comment:
consistent  *pt-table-checksum* -> remove the extra space

Reply:
Done.